vscode-lean4/src/leanclient.ts

```diff
@@ -270,7 +271,7 @@ export class LeanClient implements Disposable {
         )
         this.patchConverters(this.client.protocol2CodeConverter, this.client.code2ProtocolConverter)
         try {
-            this.client.onDidChangeState((s) =>{
+            this.client.onDidChangeState(async (s) =>{
```

Comment:
this is probably due to the vscode version change, please revert this since Gabriel wants this to be done in a separate PR.